examples/ui/client-capabilities/interaction/toolbar.js

```diff
@@ -0,0 +1,30 @@
+import { CodeSection } from '../../documentationBase';
+import useIsBrowser from '@docusaurus/useIsBrowser';
+
+export default function ToolbarDemo({ children, color }) {
+
+	const isBrowser = useIsBrowser();
+
+	if (isBrowser) {
+		const RapidImports = require('../../rapidImports');
+		RapidImports.registerComponents();
+	}
+
+	return (
+		<CodeSection>
+			<rapid-toolbar>
```

Comment:
can we have an example or horizontal and vertical position, and start and end slots and with examples too